File: OOP/Customer App/src/com/techlabs/customer/Customer.java
package com.techlabs.customer;

public class Customer {

	private int id;
	private String firstName;
	private String lastName;
	private static int nextId = 1000;
	
	public Customer(String firstName, String lastName) {
		this.firstName = firstName;
		this.lastName = lastName;
		nextId++;
		id = nextId;
	}

	public int getId() {
		return id;
	}

	public String getFirstName() {
		return firstName;
	}

	public String getLastName() {
		return lastName;
	}
}

File: Node Js/ImpExpClass/Customer.ts
export class Customer {

    constructor(private _firstName : string, private _lastName : string){
    
    }

    public get lastName(): string {
        return this._lastName;
    }
    public set lastName(value: string) {
        this._lastName = value;
    }

    public get firstName() : string{
        return this._firstName;
    }

    public set firstName(firstName : string) {
        this._firstName = firstName;
    }
}

